parking_permits/forms.py

```diff
@@ -276,11 +279,17 @@ def filter_queryset(self, qs):
             if q.isdigit():
                 query = Q(id=int(q))
             else:
-                query = (
-                    Q(customer__first_name__icontains=q)
-                    | Q(customer__last_name__icontains=q)
-                    | Q(permits__vehicle__registration_number=q)
-                    | Q(customer__national_id_number=q)
+                query = functools.reduce(
+                    operator.and_,
+                    [
+                        Q(customer__first_name__icontains=item)
+                        | Q(customer__last_name__icontains=item)
+                        | Q(customer__permits__vehicle__registration_number=item)
+                        | Q(customer__national_id_number=item)
+                        | Q(permits__vehicle__registration_number=item)
+                        | Q(permits__temp_vehicles__vehicle__registration_number=item)
+                        for item in q.split()[:6]
```

Comment:
Arbitrary limit of six tokens in the search: this can be any number, but should be limited to prevent generating a very slow query.

Reply:
@danjacob-anders Instead of hardcoded `6`, could this be a constant or variable to increase code readability?

Reply:
Added constant to Form class.